src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -446,6 +447,75 @@ public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestP
                 repoName,
                 id);
         }
+        public async Task<IList<Commit>> GetPullRequestCommitsAsync(string pullRequestUrl)
+        {
+            (string accountName, string projectName, string repoName, int id) = ParsePullRequestUri(pullRequestUrl);
+            VssConnection connection = CreateVssConnection(accountName);
+            GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
+            var prInfo = await client.GetPullRequestAsync(repoName, id, includeCommits: true);
```

Comment:
I'm not sure what type "prInfo" is, don't use var for this.

Reply:
changed it to pullRequest